Public/Src/App/Deployment.InBoxSdks.dsc

```diff
@@ -52,6 +52,18 @@ function createSdkDeploymentDefinition(serverDeployment: boolean) : Deployment.D
                             ])
                         ],
                     },
+                    {
+                        subfolder: "Sdk.Symbols",
+                        contents: [
+                            ...addIfLazy(!serverDeployment && !BuildXLSdk.isTargetRuntimeOsx, () => [
+                                importFrom("BuildXL.Tools.SymbolDaemon").withQualifier({
+                                    configuration: qualifier.configuration,
+                                    targetFramework: "net472",
+                                    targetRuntime: "win-x64"
+                                }).deployment

```

Comment:
since this is shared between drop, perhaps we should create a variable `const vstsQualifier = ...` and reuse?

Reply:
I don't want to couple them at this point

---
In reply to: [313486073](https://github.com/microsoft/BuildXL/pull/727#discussion_r313486073) [](ancestors = 313486073)